astropy/visualization/wcsaxes/tests/test_misc.py

```diff
@@ -446,3 +449,36 @@ def test_time_wcs(time_spectral_wcs_2d):
     # with a time axis.
 
     plt.subplot(projection=time_spectral_wcs_2d)
+
+
+@pytest.mark.skipif('TEX_MISSING or MATPLOTLIB_LT_21')
```

Comment:
It is not clear to me why this failed either. What if we lose the quotes?
```suggestion
@pytest.mark.skipif(TEX_MISSING or MATPLOTLIB_LT_21)
```